Userland/cut.cpp

```diff
@@ -64,15 +87,17 @@ static void expand_list(Vector<String>& tokens, Vector<Index>& indexes)
         }
 
         if (token == "-") {
-            fprintf(stderr, "cut: invalid range with no endpoint: %s\n", token.characters());
+            fprintf(stderr, "cut: invalid range with no endpoint: %s\n",
+                token.characters());
```

Comment:
There seems to be a lot of unrelated formatting changes like this. Are you running an old `clang-format` perhaps?

Reply:
Maybe, I did ran it and it messed with my enum. I didn't do anything by hand.